### PR TITLE
refactor: update react-science

### PR DESCRIPTION
**Pull request by hamed-musallam**
waiting for - https://github.com/zakodium-oss/react-science/issues/590

- https://github.com/cheminfo/nmrium/issues/2792
- https://github.com/cheminfo/nmrium/issues/2791

### CODE REVIEW

File: src/component/modal/aboutUs/AboutUsModal.tsx [hunk 1]
```diff
@@ -96,20 +96,20 @@ function AboutUsModal() {
...
       <Toolbar.Item
         onClick={openDialog}
-        titleOrientation="horizontal"
         id="logo"
         title="About NMRium"
-      >
-        <div
+        icon={<SvgLogoNmrium />}
+      />
+      {/* <div
```
Comment (targos) [inline src/component/modal/aboutUs/AboutUsModal.tsx:103]: Comments to remove?

Comment (hamed-musallam) [reply src/component/modal/aboutUs/AboutUsModal.tsx:103]: yes, it will be removed

File: test-e2e/NmriumPage.ts [hunk 2]
```diff
@@ -36,7 +36,7 @@ export default class NmriumPage {
   }
 
   public async clickTool(id: string) {
-    await this.page.click(`_react=ToolbarItem[id="${id}"]`);
+    await this.page.click(`_react=ToolbarItem[id="${id}"] >> nth=0`);
```
Comment (targos) [inline test-e2e/NmriumPage.ts:39]: You have multiple items with the same id? That feels wrong.

Comment (hamed-musallam) [reply test-e2e/NmriumPage.ts:39]: It is not the case, but for a reason, I do not fully understand the `_react=` selector retrieves two elements. I realized that this situation occurs when the tooltip is visible. For now, I addressed it this way to fix the other issues, and I plan to return to resolve it. Do you have any suggestions?

image

Comment (hamed-musallam) [reply test-e2e/NmriumPage.ts:39]: Do you have any suggestions to resolve this issue?

**Pull request merged**